### DISCUSSION TITLE
请问这些小语种模型在哪下载？

### DISCUSSION THREAD

**Discussion by Dandelion111** [PaddlePaddle/PaddleOCR#12181] [Q&A] [answered]
<img width="960" alt="9fa8c54f9a8986797a907a6d7605e31" src="https://user-images.githubusercontent.com/48303408/138264821-0204097d-807a-4a8f-984d-8e3da23f33d6.png">

**Comment by littletomatodonkey** [comment]
https://github.com/PaddlePaddle/PaddleOCR/blob/release%2F2.3/doc/doc_ch/models_list.md#%E5%A4%9A%E8%AF%AD%E8%A8%80%E8%AF%86%E5%88%AB%E6%A8%A1%E5%9E%8B

**Comment by Dandelion111** [comment]
> https://github.com/PaddlePaddle/PaddleOCR/blob/release%2F2.3/doc/doc_ch/models_list.md#%E5%A4%9A%E8%AF%AD%E8%A8%80%E8%AF%86%E5%88%AB%E6%A8%A1%E5%9E%8B

你好，这里面没有西班牙语？
![image](https://user-images.githubusercontent.com/48303408/138270808-bb2cb4c5-6683-4555-b5e2-4c1a39bd84a0.png)
是不是没有提供训练好的西班牙语模型，支持80种语言是啥意思？是训练了80种语言的模型吗？

**Comment by tink2123** [comment, marked as answer]
多语言模型是按语系训练的，字母相同的语言共享一个字典。支持80种语言就是可以预测80个语种，推荐使用whl包的方式测试：
```
pip install "paddleocr>=2.0.6"
paddleocr --image_dir /your/img/path --lang=es
```
如果希望自行训练或用源码预测的话，西班牙语属于拉丁语系，对应的
配置文件：
rec_latin_lite_train.yml
训练模型：
https://paddleocr.bj.bcebos.com/dygraph_v2.0/multilingual/latin_ppocr_mobile_v2.0_rec_train.tar
推理模型：
https://paddleocr.bj.bcebos.com/dygraph_v2.0/multilingual/latin_ppocr_mobile_v2.0_rec_infer.tar

**Comment by Dandelion111** [comment]
> 多语言模型是按语系训练的，字母相同的语言共享一个字典。支持80种语言就是可以预测80个语种，推荐使用whl包的方式测试：
> 
> ```
> pip install "paddleocr>=2.0.6"
> paddleocr --image_dir /your/img/path --lang=es
> ```
> 
> 如果希望自行训练或用源码预测的话，西班牙语属于拉丁语系，对应的 配置文件： rec_latin_lite_train.yml 训练模型： https://paddleocr.bj.bcebos.com/dygraph_v2.0/multilingual/latin_ppocr_mobile_v2.0_rec_train.tar 推理模型： https://paddleocr.bj.bcebos.com/dygraph_v2.0/multilingual/latin_ppocr_mobile_v2.0_rec_infer.tar

好的，非常感谢大佬

**Comment by benzg500** [comment]
泰语属于什么语系啊 这里没有直接的泰语识别
